software/database/src/main/java/org/apache/brooklyn/entity/database/mysql/MySqlSshDriver.java

```diff
@@ -220,6 +220,10 @@ protected void copyDatabaseConfigScript() {
 
     protected boolean copyDatabaseCreationScript() {
         String creationScriptContents = DatastoreMixins.getDatabaseCreationScriptAsString(entity);
+
+        if (creationScriptContents==null)
+            creationScriptContents = processTemplate(entity.getConfig(MySqlNode.TEMPLATE_CREATION_SCRIPT));
```

Comment:
Why not move it a level up in `DatastoreMixins.getDatabaseCreationScriptAsString` so all DBs can take advantage of the change. Otherwise looks good.